Makefile

```diff
@@ -34,6 +34,28 @@ dev-setup:
 dev-setup-extensions:
 	@./hack/dev-setup-extensions
 
+.PHONY: dev-setup-local-garden
+dev-setup-local-garden:
+	# Remove old containers and create the docker user network
+	@-docker rm -f g-etcd etcd kube-apiserver kube-controller-manager
+	@-docker network create gardener-dev
+
+    # Start the nodeless kubernetes environment
+	@./hack/local-garden/run-kube-etcd
+	@./hack/local-garden/run-kube-apiserver
+	@./hack/local-garden/run-kube-controller-manager
+
+	# This etcd will be used to storge gardener resources (e.g., seeds, shoots)
+	@./hack/local-garden/run-gardener-etcd
+
+	# Applying proxy RBAC for the extension controller
+	# After this step, you can start using the cluster at hack/local-garden/kubeconfigs/admin.conf
+	@./hack/local-garden/apply-proxy-rbac
+
+	# Now you need to run ./hack/dev-setup-register-gardener
```

Comment:
I guess you still have to create the garden Namespace yourself, right?
Can you automate that also or add a line for it?

Reply:
Can do, but the rest of the steps are left up for the developer.

Reply:
Thanks, this is easier now 👍